guides/common/modules/proc_installing-flatpak-applications-on-project-hosts.adoc

```diff
@@ -6,10 +6,13 @@
 Use the command line to install selected applications from the enabled Flatpak remotes.
 
 .Prerequisites
-* Install Flatpak.
+* Flatpak is installed.
 ifdef::satellite[]
-* Enable the {Team} Flatpak remote.
+* The {Team} Flatpak remote is enabled.
 endif::[]
+* Ensure that Podman is installed on the host.
+* To install Flatpak applications from {Project}, ensure that your {Project} account has the `view_lifecycle_environments`, `view_products`, and `view_content_views` permissions, unless the lifecycle environment allows unauthenticated pull.
+* Your {Project} account has a role that grants the `create_personal_access_tokens` permission to generate an access token for authentication.
```

Comment:
I'm still wondering if the prerequisite should say that the user must already have create the access token + link to a section that says how. Please, verify with a SME.